.circleci/config.yml

```
@@ -27,6 +27,15 @@ jobs:
           command: |
             if [ "${CIRCLE_BRANCH}" == "master" ]; then
               cd ~/cumulus-message-adapter
+
+              export VERSION=$(jq --raw-output .version package.json)
+              curl -H \
+                "Authorization: token $GITHUB_TOKEN" \
+                -d "{\"tag_name\": \"v$VERSION\", \"target_commitsh\": \"v$VERSION\", \"name\": \"v$VERSION\"}" \
```

```suggestion
                -d "{\"tag_name\": \"v$VERSION\", \"target_commitish\": \"v$VERSION\", \"name\": \"v$VERSION\"}" \
```
Typo in `target_commitish`, but is it necessary? GH API docs suggest that if the tag already exists it is unneeded. Does the tag already exist, since the value of the above value is just `v$VERSION` anyway?